assets/src/edit-story/components/form/text.js

```diff
@@ -106,79 +106,91 @@ Clear.propTypes = {
   children: PropTypes.node,
 };
 
-function TextInput({
-  className,
-  onBlur,
-  onChange,
-  onKeyDown,
-  label,
-  value,
-  flexBasis,
-  disabled,
-  clear,
-  clearIcon,
-  showClearIconBackground,
-  placeholder,
-  ...rest
-}) {
-  const { suffix } = rest;
-  const inputRef = useRef();
-  const isMultiple = value === MULTIPLE_VALUE;
-  value = isMultiple ? '' : value;
-  placeholder = isMultiple ? MULTIPLE_DISPLAY_VALUE : placeholder;
-
-  const onClear = useCallback(() => {
-    onChange('');
-    if (onBlur) {
-      onBlur({ onClear: true });
-    }
-    // Return focus to text input as otherwise focus will revert to current selection
-    inputRef.current?.focus();
-  }, [onChange, onBlur]);
-
-  return (
-    <Container
-      className={`${className}`}
-      flexBasis={flexBasis}
-      disabled={disabled}
-      suffix={suffix}
-    >
-      {/* type="text" is default but added here due to an a11y-related bug. See https://github.com/A11yance/aria-query/pull/42 */}
-      <StyledInput
-        type="text"
-        ref={inputRef}
-        placeholder={placeholder}
-        label={label}
-        value={value}
+const TextInput = forwardRef(
+  (
+    {
+      className,
+      onBlur,
+      onChange,
+      onKeyDown,
+      label,
+      value,
+      flexBasis,
+      disabled,
+      clear,
+      clearIcon,
+      showClearIconBackground,
+      placeholder,
+      ...rest
+    },
+    ref
+  ) => {
+    const { suffix } = rest;
+    const inputRef = useRef();
+    const isMultiple = value === MULTIPLE_VALUE;
+    value = isMultiple ? '' : value;
+    placeholder = isMultiple ? MULTIPLE_DISPLAY_VALUE : placeholder;
+
+    const onClear = useCallback(() => {
+      onChange('');
+      if (onBlur) {
+        onBlur({ onClear: true });
+      }
+      // Return focus to text input as otherwise focus will revert to current selection
+      inputRef.current?.focus();
+    }, [onChange, onBlur]);
+
+    return (
+      <Container
+        className={`${className}`}
```

Comment:
😬 I know this wasn't you, but since we're here should we just change this to
```suggestion
        className={className}
```